lib/html-min.cls.php

```diff
@@ -26,6 +26,7 @@ class HTML_MIN
 	 * @var boolean
 	 */
 	protected $_jsCleanComments = true;
+	protected $_skipComments = [];
```

Comment:
Please keep in mind we should keep php v5.3 working

Reply:
Yes, you are right